core/domain/html_cleaner.py

```diff
@@ -252,11 +258,12 @@ def wrap_with_siblings(tag, p):
 ]
 
 
-def convert_to_textangular(html_data):
+def convert_to_textangular(html_data, unused_exp_id=''):
```

Comment:
We need to do this, because we decided to use a common function (conversion_function) for all these content migration functions,  so we need to have a common type of definition of the function. Otherwise it will lead to a lot of duplication of code.

Reply:
No need, now. I have used partial instead.